### PR TITLE
PR message for PRs that are also fixing

### PR DESCRIPTION
**Pull request by abelsiqueira**
**Description**

Variation of PR message.

**Related issues**:

**Instructions to review the pull request**

Review online.

### CODE REVIEW

File: pr-text-fix.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Pull Request title: CITATION.cff fix and automatic validation of your citation metadata
+
+Hello!
+
+We noticed that your CITATION.cff had a small issue and fixed it - possibly by just updating `cffversion` to 1.2.0.
```
Comment (fdiblen) [inline pr-text-fix.md:5]:
```suggestion
We noticed that your `CITATION.cff` had a small issue and fixed it.
```

File: pr-text-fix.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Pull Request title: CITATION.cff fix and automatic validation of your citation metadata
```
Comment (fdiblen) [inline pr-text-fix.md:1]:
```suggestion
Pull Request title: Fix CITATION.cff and add automatic validation of your citation metadata
```

**Review by fdiblen — CHANGES_REQUESTED** [review]
suggested minor changes

**Comment by abelsiqueira** [comment]
Thanks for the review

**Review by fdiblen — APPROVED** [review]
LGTM. :game_die:

**Pull request merged**